synapse/pipeline.yml

```diff
@@ -288,24 +288,27 @@ steps:
   #
   # Our tests have three dimensions:
   #   1. Topology (Monolith, Workers, Workers w/ Redis)
-  #   2. Database (SQLite or PostgreSQL)
+  #   2. Database (SQLite, PostgreSQL)
   #   3. Python Version
   #
+  # Tests can run against either a single or multiple PostgreSQL databases.
```

Comment:
technically Synapse can run against a sqlite and a postgres, though doing that seems slightly insane and the sytest setup scripts have no support for such a thing.